examples/image-test-workflow/versions/0.0.1/workflowTemplate.yaml

```diff
@@ -0,0 +1,172 @@
+                - name: GIT_TOKEN_SECRET
+                  value: '{{ inputs.parameters.GIT_TOKEN_SECRET }}'
+                - name: DOCKERFILE
+                  value: '{{ inputs.parameters.DOCKERFILE }}'
+                - name: CONTEXT
+                  value: '{{ inputs.parameters.CONTEXT }}'
+                - name: TAG
+                  value: '{{ inputs.parameters.TAG }}'
+
+          # create mysql service container
+          - name: create-service-container
+            dependencies: [build-docker-image]
+            template: mysql-service
+            arguments:
+              parameters:
+                - name: MYSQL_SECRET
+                  value: '{{ inputs.parameters.MYSQL_SECRET }}'
+                - name: MYSQL_DATABASE
+                  value: '{{ inputs.parameters.MYSQL_DATABASE }}'
+                - name: MYSQL_HOST
+                  value: '{{ inputs.parameters.MYSQL_HOST }}'
+
+          # intialize mysql service container and test
+          - name: initialize-service-container
+            dependencies: [create-service-container]
+            template: mysql-client
+            arguments:
+              parameters:
+                - name: IMAGE
+                  value: '{{ inputs.parameters.IMAGE_NAME }}:{{ inputs.parameters.TAG }}'
+
+    - name: mysql-service
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Build service container for MYSQL'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/image-test-workflow/assets/icon.svg"
+          argo-hub-template/icon_background: "#f4f4f4"
+      inputs:
+        parameters:
+          - name: MYSQL_SECRET
+            default: mysql-secret
+          - name: MYSQL_DATABASE
+          - name: MYSQL_HOST
+      daemon: true
+      container:
+        image: mysql:8.0 
+        readinessProbe:
+          httpGet:
+            path: /
+            port: 3306
+        env: 
+          - name: MYSQL_ROOT_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: '{{ inputs.parameters.MYSQL_SECRET }}'
+                key: root-password
+          - name: MYSQL_USER
+            valueFrom:
+              secretKeyRef:
+                name: '{{ inputs.parameters.MYSQL_USER }}'
+                key: user
+          - name: MYSQL_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: '{{ inputs.parameters.MYSQL_PASSWORD }}'
+                key: password
+          - name: MYSQL_DATABASE
+            value: '{{ inputs.parameters.MYSQL_DATABASE }}'
+          - name: MYSQL_HOST
+            value: '{{ inputs.parameters.MYSQL_HOST }}'
+
+    - name: mysql-client
+      metadata:
+        annotations:
+          argo-hub-template/description: 'Initialize service container and test image'
+          argo-hub-template/icon_url: "https://github.com/codefresh-io/argo-hub/blob/main/examples/image-test-workflow/assets/icon.svg"
```

Comment:
need to be change to this "https://cdn.jsdelivr.net/gh/codefresh-io/argo-hub@main/examples/image-test-workflow/assets/icon.svg"